python/ray/data/_internal/stats.py

```diff
@@ -549,6 +582,34 @@ def _run_update_loop():
 
     # Execution methods
 
+    def _generate_per_node_metrics(
```

Comment:
This method seems to be doing the opposite of its name -- aggregating per-node metrics